docs/simple_switch.md

```diff
@@ -25,6 +25,24 @@ separate executable program named `psa_switch`, separate from the
 This document aims at providing P4 programmers with important information
 regarding the simple_switch architecture.
 
+
+## Differences between simple_switch and simple_switch_grpc processes
+
+The simple_switch process was developed first, and supports a
+Thrift-based control API that was custom developed for BMv2, and to
+our knowledge is not supported by other switches, unless they are
+based on simple_switch.
+
+The simple_switch_grpc process was developed based on simple_switch,
+and supports both the P4Runtime API, as a server/switch, and the
+simple_switch Thrift API.  The Thrift API is still supported because
```

Comment:
This new section is great, but I think we should make it sure that Thrift is _not_ available by default in simple_switch_grpc to avoid confusion: https://github.com/p4lang/behavioral-model/tree/main/targets/simple_switch_grpc#enabling-the-thrift-server

Reply:
Eventually you will teach me to say less, rather than more incorrect things, in my documentation, but not today :-). Thanks for the correction.